pandas/core/frame.py

```diff
@@ -1374,31 +1377,20 @@ def to_records(self, index=True, convert_datetime64=None):
         rec.array([('a', 1, 0.5 ), ('b', 2, 0.75)],
                   dtype=[('index', 'O'), ('A', '<i8'), ('B', '<f8')])
 
+        If the DataFrame index has no label then the recarray field name
+        is set to 'index'. If the index has a label then this is used as the
+        field name:
+
+        >>> df.index.rename("I", inplace=True)
```

Comment:
don't use inplace in examples, pls just reassign the index

Reply:
Done!